py/torch_tensorrt/dynamo/conversion/impl/slice/ops.py

```diff
@@ -37,10 +37,6 @@ def slice_op(  # TODO: This should be slice not whatever is in base
             "of the TensorRT region!"
         )
 
-    dim = get_positive_dim(dim, len(input.shape))
-    start = get_positive_dim(start, input.shape[dim])
-    stop = get_positive_dim(stop, input.shape[dim])
-
```

Comment:
I did this change because in some cases I want to use `slice_op` to get a reserved tensor but the `stop` arg cannot be negative. For example, when doing:
```
impl.slice.slice_op(
                ctx,
                target,
                source_ir,
                f"{name}_slice_post{i}",
                input,
                dim=dim,
                start=2,
                stop=-1,
                step=-1,
            )
```
if has these lines, `stop` would be a positive value (i.e. `length - 1`), but since `step` is still negative, it will never stop, however, my expected sequence is `[2, 1, 0]`. 

The reason why I don't use positive `stop` and `step` is I wanna get a reversed tensor.

The unit tests proved these lines are not necessary.

Reply:
The issue with removing this is that the line `output_shape[dim] = math.ceil((stop - start) / step)` below will be incorrect when certain values are negative. This change would have to be accompanied by a fix for that equation, since these test cases should otherwise fail:
https://github.com/pytorch/TensorRT/blob/867dc7bf23b34f8d3809a1d64841b0013a9a0054/tests/py/dynamo/conversion/test_slice_aten.py#L16-L17

Reply:
Thanks for the comments! I undid the change due to better implementation.